modules/river/hm.nix

```diff
@@ -0,0 +1,18 @@
+{ config, lib, ... }:
+
+with config.stylix;
+with config.lib.stylix.colors;
```

Comment:
Consider removing these "global" `with`s to align with https://github.com/danth/stylix/pull/425.

Reply:
Done